stardog/admin.py

```diff
@@ -46,37 +45,45 @@ def shutdown(self):
         """
         self.client.post('/admin/shutdown')
 
-    #TODO
-    # def alive(self):
-    #     """
-    #     Determine whether the server is running
-    #     https://stardog-union.github.io/http-docs/#operation/aliveCheck
-    #     :return:
-    #     """
+    def alive(self):
+        """
+        Determine whether the server is running
+        https://stardog-union.github.io/http-docs/#operation/aliveCheck
+        """
+        r = self.client.get('/admin/alive')
+        alive = True if r.status_code == 200 else False
+        return alive
 
-    #TODO:
-    # def healthcheck(self):
-    #     """
-    #     Determine whether the server is running and able to accept traffic
-    #     https://stardog-union.github.io/http-docs/#operation/healthCheck
-    #     :return:
-    #     """
 
-    #TODO
-    # def get_prometheus_metrics(self):
-    #     """
-    #     Return metric information from the registry in Prometheus format
-    #     https://stardog-union.github.io/http-docs/#operation/prometheus
-    #     :return:
-    #     """
+    def healthcheck(self):
+        """
+        Determine whether the server is running and able to accept traffic
+        https://stardog-union.github.io/http-docs/#operation/healthCheck
+        """
+        r = self.client.get('/admin/healthcheck')
+        health = True if r.status_code == 200 else False
+        return health
```

Comment:
cut out the middleman 
```suggestion
        return r.status_code == 200
```